python/ray/data/tests/test_streaming_executor.py

```diff
@@ -257,11 +257,11 @@ def test_dispatch_next_task():
 
     o2.add_input = MagicMock()
     op_state.dispatch_next_task()
-    assert o2.add_input.called_once_with(ref1)
+    o2.add_input.assert_called_once_with(ref1)
```

Comment:
```suggestion
    o2.add_input.assert_called_once_with(ref1, input_index=0)
```
Same for the other one.